packages/tools/viewer/src/customViewerElement.ts

```diff
@@ -0,0 +1,29 @@
+import type { CanvasViewerOptions } from "./viewerFactory";
+import type { TransformNode } from "core/Meshes";
+import { customElement } from "lit/decorators.js";
+import { Viewer } from "./viewer";
+import { ViewerElement } from "./viewerElement";
+
+// TODO: DELETE THIS FILE

```

Comment:
is this still a valid comment?

Reply:
I must have messed up a merge and missed it somehow. This is definitely supposed to be deleted.